public/app/features/integrated-alerting/components/AlertRules/AlertRulesActions/AlertRulesActions.messages.ts

```diff
@@ -1,4 +1,7 @@
 export const Messages = {
   getEnabledMessage: (summary: string) => `Alert rule "${summary}" successfully enabled`,
   getDisabledMessage: (summary: string) => `Alert rule "${summary}" successfully disabled`,
+  createSuccess: 'Alert rule created',
+  deleteSuccess: 'Alert rule deleted',
```

Comment:
This should have the name of the rule deleted similar  to the enable/disable message

Reply:
Done